api/src/main/java/com/github/bsideup/liiklus/records/HeaderMapper.java

```diff
@@ -0,0 +1,51 @@
+package com.github.bsideup.liiklus.records;
+
+import io.cloudevents.v1.ContextAttributes;
+import lombok.NonNull;
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import static com.github.bsideup.liiklus.records.AttributeMapper.HEADER_PREFIX;
+
+/**
+ * See https://github.com/cloudevents/spec/blob/v1.0/kafka-protocol-binding.md
+ */
+class HeaderMapper {
+
+    static final String CONTENT_TYPE = "content-type";
+
+    public static Map<String, String> map(
+            @NonNull Map<String, String> attributes,
+            @NonNull Map<String, String> extensions
+    ) {
+        Map<String, String> result = new HashMap<>();
+
+        extensions.forEach((key, value) -> {
```

Comment:
This is a NO-OP

Reply:
there is `result.put` for all non-null values, unless I am missing something :)

Reply:
Ah, my bad, I thought this was acting in place (which would cause concurrent modification anyway). So my other comment is moot too.